engine/ingame/view.py

```diff
@@ -43,10 +40,16 @@ def __init__(
             viewport_size[1],
         )
 
-    def setup(self) -> None:
-        """Sets up the view."""
-        self.width = self.game_world.width * self.game_world.tile_width
-        self.height = self.game_world.height * self.game_world.tile_height
+    @property
+    def world_width(self) -> int:
```

Comment:
Lint is complaining about missing docstrings for these two properties, you'll need to add them.